components/Board.tsx

```diff
@@ -24,9 +24,12 @@ const GridContainer = styled.div`
   flex-direction: column;
 `;
 
-const BoardTitle = styled.h3`
+const BoardTitle = styled.div`
   margin-bottom: 6px;
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
I think instead of setting the direction to row reverse, you can just reverse the order of the items in the html

Reply:
Ha I've contemplated the best approach. I guess I'm not used to rtl, so usually I'd rather write markup to support ltr and "fix" it in the css if needed otherwise, here its probably best to support rtl first so I'll change it no problem.